apps/vr-tests-react-components/src/stories/Toast/Toast.stories.tsx

```diff
@@ -241,3 +241,51 @@ export const TitleOnlyInverted = () => {
 
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=29678&commitId=907339ef8219ca8e41e4c1f20c2b5f6613cf7f89&env=prod&clientType=FLUENTUI)


  <details><summary><b>Toast</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Toast.Toast Body Word Break.Toast visible.chromium.png|0| Added|
|Toast.Toast Body Word Break.chromium.png|0| Added|
|Toast.Toast Title Word Break.Toast visible.chromium.png|0| Added|
|Toast.Toast Title Word Break.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>